build.sh

```diff
@@ -33,4 +33,12 @@ rm mantle -rf
 
 dnf remove -y cargo golang
 rpm -q grubby && dnf remove -y grubby
+
+# more tooling for building openshift/os in the container
+cd /root
+git clone https://github.com/openshift/os
```

Comment:
I'm not sure about having a (totally conceptual) circular dependency between the repos *just* for GPG keys.   I'd been thinking that this coreos-assembler container can be used conveniently in a "BYO" build scenario - it could be used for Silverblue or people doing custom CoreOS-style IoT builds, etc.

Let's just do the GPG key copying in that repo?

Reply:
Sure, I'll drop this out.